geonode/documents/views.py

```diff
@@ -489,9 +490,30 @@ def document_metadata_detail(request, docid, template='documents/document_metada
 @login_required
 def document_list(request):
     documents = Document.objects.all()
+    filename = str(datetime.datetime.now()) + "_documents.csv"
     response = HttpResponse(content_type='text/csv')
-    response['Content-Disposition'] = 'attachment; filename="geonode_documents.csv"'
+    response['Content-Disposition'] = 'attachment; filename=' + filename
     writer = csv.writer(response)
+    # Writes header row
+    writer.writerow(['Layer ID',
+                    'Title',
+                    'Name',
+                    'Creation Date',
+                    'Hazard Type',
+                    'Supplemental Information',
+                    'Hazard Unit',
+                    'Region',
+                    'Owner Organization',
+                    'License',
+                    'Restrictions',
+                    'Document URL',
+                    'Distribution URL',
+                    'Data Quality Statement',
+                    'Is Published',
+                    'Point of Contact',
+                    'Category',
+                    'Abstract'
+                    ])
     for document in documents:
         t = filter(lambda x: x in string.printable, document.title)
```

Comment:
Changed variable names to title, docid. I believe docid is more consitently used throughout the script, and id would not function properly.